### PR TITLE
Fix tslint filenames and Add option to specify tslint output as a file #91

### LINKED ISSUE #75: Pass --project and --type-check parameters to tslint
How can i pass  --project and --type-check to tslint?

### PR DESCRIPTION
**Pull request by NikitaEgorov**
Fix #75

### CODE REVIEW

**Comment by NikitaEgorov** [comment]
@Pablissimo please check my PR

**Comment by Pablissimo** [comment]
Going to be a couple days before I can, workload too high atm but shall merge when possible - thanks for the contrib!

File: src/main/java/com/pablissimo/sonar/TsLintSensor.java [hunk 1]
```diff
@@ -72,11 +72,11 @@ else if (config.getConfigFile() == null && config.getPathToTsConfig() == null) {
                 continue;
             }
 
-            String pathAdjusted = file.absolutePath().replace('\\', '/');
+            String pathAdjusted = file.absolutePath().replace('\\', '/').toLowerCase();
```
Comment (Pablissimo) [inline src/main/java/com/pablissimo/sonar/TsLintSensor.java]: What's the deal here? If you're on Linux with two files that differ only by case then the plugin'll blow up.

Comment (NikitaEgorov) [inline src/main/java/com/pablissimo/sonar/TsLintSensor.java]: @Pablissimo 
It's fix problem with ``C:/`` SonarQube return ``C:/`` but TSLint return ``c:/``

Comment (Pablissimo) [inline src/main/java/com/pablissimo/sonar/TsLintSensor.java]: So I can only get tslint to return the same casing as I supply it, i.e. if I give it a file with a lower-case drive letter it spits it back lower-case, while if I give it one upper-case it appeared to keep it upper-case. Will try and repro tonight

**Pull request merged**